binnacle/lib/DataModel.dart

```diff
@@ -0,0 +1,15 @@
+class DataModel{
+  double windSpeed;
+  double windDirection;
+  double idealHeading;
+  double idealBoomPosition;
+  double compassHeading;
+
+  DataModel(){
+    windSpeed = 0;
```

Comment:
I'm cautious about this, as I like the idea of just going negative (such as -90 = 270). However, that's math notation, and I think we could compromise on it for the sake of better design.

Reply:
i think, they are by default null and I think that is enough to say for them not being initialized. I don't think they are initialized to 0 in Dart. For example, in main.dart, _direction is uninitialized and on ios, which doesn't trigger the compass, the value is null because it's caught by that ternary check in the widget.

Reply:
https://www.dartlang.org/guides/language/language-tour#default-value